plugin/docker/volume.go

```diff
@@ -226,6 +227,58 @@ func (v *volume) VolumeStream(ctx context.Context, path string) (io.ReadCloser,
 	return plugin.CleanupReader{ReadCloser: output, Cleanup: killAndDelete}, nil
 }
 
+func (v *volume) VolumeDelete(ctx context.Context, path string) (bool, error) {
+	// Create a container that mounts a volume and deletes its file. Run rm -rf on it.
```

Comment:
This is almost entirely a duplicate of VolumeList. Can we extract it into a common helper?

A bit more work would be to extract common elements of all the Volume* commands here into a helper struct that we can safely cleanup.